Lib/datetime.py

```diff
@@ -381,6 +405,40 @@ def _check_utc_offset(name, offset):
                          "-timedelta(hours=24) and timedelta(hours=24)" %
                          (name, offset))
 
+
+def _check_int_field(value):
```

Comment:
I guess you can ignore the rest of my comments on this function, because the big one is: Why did you add this function, that, as far as I can tell, is not used anywhere else in the new code?